aspnetcore/host-and-deploy/blazor/webassembly.md

```diff
@@ -416,12 +416,24 @@ dir .\_framework\_bin | rename-item -NewName { $_.name -replace ".dll\b",".bin"
 ((Get-Content .\_framework\blazor.boot.json -Raw) -replace '.dll"','.bin"') | Set-Content .\_framework\blazor.boot.json
 ```
 
+If service worker assets are also in use on Windows, add the following command:
```

Comment:
This `on Windows` confused me. Wondering whether we should have `On Windows` as a subsection - so all the content is visually under it.

```suggestion
If service worker assets are also in use, add the following command:
```